meinberlin/assets/scss/components/_moderation_info.scss

```diff
@@ -0,0 +1,58 @@
+.moderation-info__accordion-title {
+    display: block;
+    position: relative;
+    padding: 0.8 * $padding 1.7 * $padding 0.8 * $padding $padding;
+    text-decoration: none;
+    color: inherit;
+
+    .moderation-info__accordion-chevron {
+        float: right;
+        transition: transform 0.3s;
+        line-height: inherit;
+    }
+
+    a[aria-expanded="true"] .moderation-info__accordion-chevron {
+        transform: rotate(180deg);
+    }
+}
+
+.moderation-info__accordion-body {
+    padding: 0 $padding $padding $padding;
+}
+
+.moderation-info__title {
+    display: inline-block;
+    margin: 0;
+}
+
+.moderation-info__btn {
+    float: right;
```

Comment:
float is why the padding is uneven in the title element, I would try to use flexbox here and as it doesn't remove the element from normal page flow so the padding on the surround title element will still work.